js/interface.js

```diff
@@ -390,6 +392,35 @@ function trimColumns(columns) {
   });
 }
 
+function getEmptyColumns(columns, entries) {
+  var emptyColumns = _.filter(columns, function(column) {
+    return emptyColumnNameRegex.test(column);
+  });
+
+  _.forEach(entries, function(entry) {
+    for (var i = emptyColumns.length - 1; i >= 0; i--) {
```

Comment:
what is this for cycle supposed to do?

Reply:
This cycle is used to iterate through `emptyColumns` (columns that have the `Column (X)` format) for each entry and check if column has a value.
When column has value in some entry - then we remove this column from `emptyColumns`.
This cycle is descending to remove items from `emptyColumns` without index manipulations.
Also I wanted to use Handsontable's `isEmptyCol` but it only checks `null` and `undefined` values (not empty strings).

UPD: I added check at the beginning of `forEach` if `emptyColumns` is empty to stop iterating through all entries.